databases/recutils/Portfile

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                recutils
+version             1.8
+
+categories          databases
+maintainers         @zaid openmaintainer
+description         Tools to work with human-editable, plain text data files
+long_description    \
+    GNU Recutils is a set of tools and libraries to access human-editable, \
+    plain text databases called recfiles. The data is stored as a sequence \
+    of records, each record containing an arbitrary number of named fields.
+
+platforms           darwin
+license             GPL-3+
+
+homepage            https://www.gnu.org/software/recutils/
+master_sites        https://ftp.gnu.org/gnu/recutils/
+mirror              https://ftpmirror.gnu.org/gnu/recutils/
+checksums           sha256  df8eae69593fdba53e264cbf4b2307dfb82120c09b6fab23e2dad51a89a5b193 \
+                    rmd160  d3f0042a5898ad5630f1295d9fa2f87a08031a34 \
+                    size    2474024
+
+depends_lib         port:libgcrypt \
+                    port:ossp-uuid \
+                    port:flex \
+                    port:check
```

Comment:
Good point 👍🏼 I just added this to the portfile.

Reply:
one more thing: are the ports "check" and "flex" needed at build and runtime? For example, I would think "check" is perhaps only a "test" dependency? But I haven't tried to do a build in trace-mode (with the -t) flag without listing these to see if the build finishes. Perhaps you know or could investigate this?

Reply:
@reneeotten you're right, the `check` dependency is only needed during build-time (for running `lint test` / `make check`). Is there a way to define it as an optional dependency that's only applicable for `lint test` ?

I included `flex` since it was listed in the project's README but I'll try removing it and seeing if the build succeeds.

Reply:
Just updated the portfile with `depends_build` and `depends_test` and move both `flex` and `check` into those sections.